backend/src/packages/users/user.model.ts

```diff
@@ -22,6 +27,20 @@ class UserModel extends AbstractModel {
     return DatabaseTableName.USERS;
   }
 
+  public static override get modifiers(): Modifiers<QueryBuilder<UserModel>> {
+    return {
+      withoutPassword(builder): QueryBuilder<UserModel> {
```

Comment:
So it's only declaration but never used.

Reply:
you need to add this modifier (withoutPassword) everywhere where you execute query and expect user without password  https://vincit.github.io/objection.js/recipes/modifiers.html

Reply:
Yes, that's what I was asking about [in another thread](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/48#discussion_r1307210242), am I supposed to use it with additional entity or not now?

Reply:
you should use it where UserEntity is used, because it is without password, and you should not use it where we need UserWithPasswordEntity

Reply:
I added it.